opencensus/proto/stats/metrics/metrics.proto

```diff
@@ -68,9 +68,17 @@ message MetricDescriptor {
   }
   Type type = 4;
 
-  // TODO: Consider to add LabelInfo{key, description} if needed.
```

Comment:
Why is description needed?

Reply:
Answer: some backends make use of it?

Reply:
Yes, 1. it's useful for some backend, and 2. provides more information to human readers.